tdrs-frontend/.env

```diff
@@ -0,0 +1,38 @@
+###
+# Default environment variables used by React
+# https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env  # noqa
```

Comment:
Love the link here, all the context a dev needs to understand what's going on! 😍